src/app/pages/home-page/components/gdsc-details/gdsc-details.component.css

```diff
@@ -0,0 +1,65 @@
+.container {
+  width: 70%;
+  display: flex;
+  flex-direction: row;
+  margin: 10px auto 50px auto;
+  align-items: center;
+  justify-content: center;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.image-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+img {
+  width: 100%;
+}
+
+ul {
+  margin-left: 0;
+  padding-left: 0;
+}
+
+li {
+  display: inline-block;
+  padding-right: 15px;
+  width: 10%;
+}
+
+a:hover {
+  background: #4da5ff;
+}
+
+h1 {
+  font-family: sans-serif;
+  color: darkslategray;
+  font-weight: bold;
+  font-size: xx-large;
+}
+
+p {
+  font-size: large;
+  overflow: hidden;
+  text-align: justify;
```

Comment:
we can remove this because the text looks weird on smaller screens with only 2 words on some rows because of it